B827EBFFFE84F285.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "og_lora_gateway_citycenter",
```

Comment:
The ID should match the file name, so it should be `B827EBFFFE84F285`. Please update and then we can merge. 😊

Reply:
okay, thank you